day_two_guide/topics/proc_restoring-project.adoc

```diff
@@ -11,25 +11,31 @@ Module included in the following assemblies:
 = Restoring a project
 
 To restore a project, create the new project, then restore any exported files
-by running `oc create -f pods.json`. However, restoring a project from scratch
-requires a specific order because some objects depend on others. For
-example, you must create the `configmaps` before you create any `pods`.
+by running `oc create -f <file_name>`.
 
 [discrete]
 == Procedure
 
-. If the project was exported as a single file, import it by running the
-following commands:
+. Create the project:
++
+----
+$ oc new-project <project_name>
+----
+
+. Import the project objects:
 +
 ----
-$ oc new-project <projectname>
 $ oc create -f project.yaml
```

Comment:
I think that since all of the files from the other resource exports have the namespace set to the project that was backed up, that you have to use the same name as the project that was backed up.

I'll specify that when creating the new project in the previous step.

Reply:
@barleyer I updated. Let me know if project backup/restore looks okay to you now, or if you have any further feedback. Thanks!